includes/Core/Authentication/Authentication.php

```diff
@@ -525,11 +529,9 @@ private function inline_js_setup_data( $data ) {
 			$data['hasSearchConsoleProperty'] = false;
 		}
 
-		$reauth                        = isset( $_GET['reAuth'] ) ? ( 'true' === $_GET['reAuth'] ) : false; // phpcs:ignore WordPress.CSRF.NoNonceVerification.
-		$data['showModuleSetupWizard'] = $reauth;
+		$data['showModuleSetupWizard'] = $this->context->input()->filter( INPUT_GET, 'reAuth', FILTER_VALIDATE_BOOLEAN );
 
-		$module_to_setup       = isset( $_GET['slug'] ) ? sanitize_key( $_GET['slug'] ) : ''; // phpcs:ignore WordPress.CSRF.NoNonceVerification.
-		$data['moduleToSetup'] = $module_to_setup;
+		$data['moduleToSetup'] = sanitize_key( $this->context->input()->filter( INPUT_GET, 'slug' ) );
```

Comment:
If `slug` is not set, this would cause a problem due to `sanitize_key()` not receiving a string. We need to make sure the value is a string, an empty string would be fine.

Reply:
Updated!